packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -909,7 +914,7 @@ function renderRoot(
       return commitRoot.bind(null, root);
     }
     case RootSuspended: {
-      if (!isSync) {
+      if (!isSync && expirationTime !== Offscreen) {
```

Comment:
This should condition should never happen, right? I'm not sure this is actually the correct way of modeling it because if we did batch other work in with this level for some other reason, then you would still want to suspend.